README.md

```diff
@@ -3,37 +3,102 @@
+
+* It should work on any operating system that supports Ansible, Git, VirtualBox and Vagrant. 
+
+* It uses a suite of [Ansible](https://www.ansible.com/how-ansible-works) playbooks called within the Vagrantfile's provisioners section to install [Islandora CLAW](https://github.com/Islandora-CLAW/CLAW), its dependencies, additional software, user roles and more.
+
+* The information listed below is to serve as a quick getting started guide. 
+  * For more detailed information please navigate to the Technical documentation [here](/docs/index.md)
+
+---
 
 ## Requirements
 
-1. [VirtualBox](https://www.virtualbox.org/)
-2. [Vagrant](http://www.vagrantup.com/) 1.8.5+
-3. [git](https://git-scm.com/)
-4. [ansible](https://www.ansible.com/community) 2.3+
+1. [VirtualBox](https://www.virtualbox.org/wiki/Downloads) 5.1+
+2. [Vagrant](https://www.vagrantup.com/downloads.html) 1.95+
+3. [git](https://git-scm.com/downloads) 2.1+
+4. [ansible](http://docs.ansible.com/ansible/latest/intro_installation.html) 2.3+
+
+## System Resources
 
-## Variables
+By default the virtual machine (guest) that is built uses 3GB of RAM. The host machine **will need to be able to support** the additional memory use. 
 
-### System Resources
+One can override the CPU and RAM allocation by creating `ISLANDORA_VAGRANT_CPUS` and `ISLANDORA_VAGRANT_MEMORY` environment variables and setting the values within the Vagrantfile. 
 
-By default the virtual machine that is built uses 3GB of RAM. Your host machine will need to be able to support the additional memory use. You can override the CPU and RAM allocation by creating `ISLANDORA_VAGRANT_CPUS` and `ISLANDORA_VAGRANT_MEMORY` environment variables and setting the values. For example, on an Ubuntu host you could add to `~/.bashrc`:
+**Example 1:** Within an Ubuntu host machine one could add these values to `~/.bashrc` to increase the CPU & Memory usage of the (guest).
 
 ```bash
 export ISLANDORA_VAGRANT_CPUS=4
 export ISLANDORA_VAGRANT_MEMORY=4096
 ```
 
-## Use
+**Example 2:** Within the Vagrant file, one could edit these values to increase the CPU & Memory usage prior to running `vagrant up` to launch the virtual environment (guest) with more resources allocated.
+
+````
+$cpus   = ENV.fetch("ISLANDORA_VAGRANT_CPUS", "4")
+$memory = ENV.fetch("ISLANDORA_VAGRANT_MEMORY", "4096")
+````
+
+## System Ports
+The following services and ports are mapped/exposed from the development environment virtual machine (guest) to the (host) laptop, workstation or server that runs Virtualbox.
+
+Please note that in the event of conflict, the port values for the **host ONLY** should be changed within the Vagrantfile. These values can be arbitrary. This can often occur when running more than one Vagrant server at time. 
+
+**Original Vagrant port settings**
+* `guest: 8000, host: 8000` # **Apache**
+* `guest: 8080, host: 1234` # **Tomcat**
+* `guest: 8181, host: 8181` # **Karaf**
+* `guest: 8282, host: 8282` # **Islandora Microservices**
```

Comment:
This line can be removed

Reply:
Or more accurately, should be replaced with

`guest: 8081, host: 8081` # **Api-X**